CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ability to configure email verification for new users (<https://github.com/opencv/cvat/pull/1929>)
 - Link to django admin page from UI (<https://github.com/opencv/cvat/pull/2068>)
 - Notification message when users use wrong browser (<https://github.com/opencv/cvat/pull/2070>)
+- Annotation in process outline color wheel (<https://github.com/opencv/cvat/pull/2084>)
```

Comment:
@bsekachev , @ActiveChooN , could you please fix CHANGELOG?

Reply:
Fixed in develop